pulsar-client-cpp/python/pulsar/__init__.py

```diff
@@ -321,7 +321,8 @@ def create_producer(self, topic,
                         batching_max_messages=1000,
                         batching_max_allowed_size_in_bytes=128*1024,
                         batching_max_publish_delay_ms=10,
-                        message_routing_mode=PartitionsRoutingMode.RoundRobinDistribution
+                        message_routing_mode=PartitionsRoutingMode.RoundRobinDistribution,
+                        properties=None,
```

Comment:
We should add `properties` to the pydoc string below

Reply:
yeah will add

Reply:
done